dspace_metadata_import.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
This file might needs some tabs-vs-spaces cleanup.  The alignment of some of the comment formatting looks off for me.

Reply:
No big deal, but you still have some tabs-vs-spaces issues.